### ISSUE TITLE
!important problem within Outlook

### ISSUE THREAD

**Issue by illycz** [foundation/foundation-emails#607]
**How can we reproduce this bug?**
Insert some !important value, Outlook will filter it. (In my case, if I wont remove underlined links in some browser, I need text-decoration: none !important;)

Write out the HTML (or Inky code) that causes the issue.

**What did you expect to happen?**
Is it possible if linter don't remove identic values? So in this example: `<a href="#" style="text-decoration: none !important; text-decoration: none;">My link</a>`

**What happened instead?**
Linter remove one value

**What email clients does this happen in?**

**Comment by MaverickComputingTesting** [comment]
This is also a problem with other clients like web.de gmx.de etc. Currenlty working on a newsletter in a german company - any mobile / tablet users of said email clients see a messed up version of the email/newsletter:

when the !important rule is removed by these clients the media query rules are ignored in favor of the inline widths which mess up the column layout. 

I have to manually fix this in the source by removing the 'style="width:123px;"' and where needed (ie. on the container element add a table attribute 'width="123"' instead.   MAN - it was painful!

At the moment it makes this potentially great framework framework useless for intended tasks.

**Comment by illycz** [comment]
Yes, I also did changes manually, but after another build, everything gone, of course :)

**Comment by MaverickComputingTesting** [comment]
In your case is the `!important` rule removed from all the css - not just the inline css? And are you testing in a mobile/smaller device or on a desktop - (are the columns working in a mobile device for you?) Do you mean outlook.com or outlook software package?

**Comment by illycz** [comment]
Inliner remove one rule. Now I don't know if first or second...
So when Outlook remove rule within !Important and second rule isn't there, that's my problem...

Sorry for english

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
